testing/run-scripts/run_cloud.sh

```diff
@@ -13,40 +13,43 @@ set -e
 PREBUILT=
 NPM=true
 INVITE_CODE=
-REFRESH=false
+UPDATE=false
+WIPE=false
 PUBLIC_IP=
 BANNER=
 
 SSHD_PORT=5000
 
 function usage () {
-  echo "$0 [-p path] [-i invite code] [-r] [-d ip] [-b banner] browser-version"
-  echo "  -p: path to pre-built uproxy-lib repository"
-  echo "  -i: invite code"
-  echo "  -r: recreate Docker images (WARNING: will invalidate invite codes)"
-  echo "  -d: override the detected public IP (for development only)"
-  echo "  -b: name to use in contacts list"
-  echo "  -h, -?: this help message"
-  echo
-  echo "Example browser-version: chrome-stable, firefox-canary"
-  exit 1
+    echo "$0 [-p path] [-i invite code] [-u] [-w] [-d ip] [-b banner] browser-version"
+    echo "  -p: path to pre-built uproxy-lib repository"
+    echo "  -i: invite code"
+    echo "  -u: update Docker images (backs up invite code unless -i or -w used)"
+    echo "  -w: wipe existing images (WARNING: will invalidate invite codes)"
```

Comment:
I love this `-w` flag. Let's change "wipe existing images" to something like "do not migrate data from existing containers", to make clearer the difference between it and `-u`.